edit.php

```diff
@@ -1,92 +1,50 @@
 <?php
-{
-	mysql_connect('localhost');
-	mysql_select_db('test_db');
+
+	include('header.inc.php');
+
+	if(!isset($_GET['id']) || empty($_GET['id']) || !is_numeric($_GET['id']))
+	{
+		die("Invalid entry ID.");
+	}
+	$id = (int) $_GET['id'];
 
 	if(isset($_POST['update']))
 	{
-		$id = (int)$_GET['id'];
 		$entry = htmlspecialchars(strip_tags($_POST['entry']));
 		$title = htmlspecialchars(strip_tags($_POST['title']));
+		$password = (isset($_POST['password'])) ? 1 : 0;
```

Comment:
This is ternary notation and saves a good few lines for simple things like this.


Actually now I look at it you could well just say `$password = isset($_POST['password']);`